js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -203,44 +209,112 @@ const ValidatingController = modules.Controller.inherit((function() {
         },
 
         createValidator: function(parameters, $container) {
-            const that = this;
-            const editingController = that._editingController;
-            const column = parameters.column;
             let editData;
             let editIndex;
-            const defaultValidationResult = function(options) {
-                if(options.brokenRule) {
-                    options.brokenRule.columnIndex = column.index;
-                    options.brokenRule.column = column;
+            let visibleColumns;
+            let columnsController;
+            const editingController = this._editingController;
+            const column = parameters.column;
+            const renderCellPendingIndicator = () => {
```

Comment:
The createValidator method became big. Move inner functions outside.

Reply:
OK